src/components/ChatEntry.js

```diff
@@ -1,22 +1,73 @@
 import React from 'react';
+// import {useState} from 'react';
+import TimeStamp from './TimeStamp'
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
-const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLiked}) => {
+  //The original way I implemented getting the date, before realizing TimeStamp existed
+  // const newDate = new Date()
+  
+  // let diff = (newDate.getTime() - new Date(timeStamp).getTime()) / 1000;
+  // diff /= (60*60*24);
+  let timePassed = {
+    time: timeStamp
+  };
+  let currentYears = TimeStamp(timePassed)
+  
+  // const [buttonText, setButtonText] = useState('🤍')
+  const buttonText = liked ? '❤️' : '🤍'
+  const toggleLiked = (id, likedStatus) => {
+    updateLiked(id, likedStatus);
+  };
+  if (Number(id)%2){
```

Comment:
In order to make the code more DRY, you can do this check earlier and then save the class name in a variable. It would look like this:
```
const className = Number(id) % 2 ? "chat-entry local" : "chat-entry remote"
```
and then here you would do `div className = {className}`. That way you don't have to write all of the HTML twice.

Great job taking on this optional enhancement!